solutions/devsprint-leonardo-santos-2/FinanceApp/Components/ActivityListView.swift

```diff
@@ -62,6 +62,26 @@ extension ActivityListView {
             tableView.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor),
         ])
     }
+    
+    private func configureMenu() -> UIMenu {
+        let favorite = UIAction(
+            title: "Favoritar",
+            image: SFSymbols.heart.image,
+            identifier: nil
+        ) { _ in
+            print("Item Favoritado!")
+        }
+        
+        return UIMenu(
+            title: String(),
+            image: nil,
+            identifier: nil,
+            options: .displayInline,
+            children: [
+                favorite
+            ]
```

Comment:
Todas as propriedades do init do UIMenu já tem valores default, então a gente pode omitir os desnecessários

```suggestion
        return UIMenu(
            options: .displayInline,
            children: [favorite]
        )
```